src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -800,7 +813,7 @@ function WorkspaceMembersPage({personalDetails, route, policy, currentUserPerson
                                 shouldUseUserSkeletonView
                                 disableKeyboardShortcuts={removeMembersConfirmModalVisible}
                                 headerMessage={shouldUseNarrowLayout ? headerMessage : undefined}
-                                onSelectRow={openMemberDetails}
+                                onSelectRow={toggleOrNavigate}
```

Comment:
```suggestion
                                onSelectRow={rowSelectionHandler}
```

@ChavdaSachin  I think we can update the function name to be generic.
WDYT @srikarparsi ?